### PR TITLE
Initial tests bd

### LINKED ISSUE #31: App- integration test
Integration testing needs to be provided fo `App.js`

### LINKED ISSUE #46: Add fetch for question data from the backend/ remove dataset

### LINKED ISSUE #57: Change API calls to use hosted server instead of localhost
https://indecision-maker.herokuapp.com/api/v1/movies
https://indecision-maker.herokuapp.com/api/v1/boardGames
https://indecision-maker.herokuapp.com/api/v1/cardGames
https://indecision-maker.herokuapp.com/api/v1/movies

### PR DESCRIPTION
**Pull request by BlakeDonn**
#### What's this PR do?  

- Adjusts apiCalls to use deployed endpoints instead of local
- Adds tests for App, Homepage, and BrowsePage

#### Where should the reviewer start? 

- Unskip all tests and check they pass (last two in app should fail until we connect functionality"

#### How should this be manually tested?  

- Check all tests are passing and boot up app, ensure all endpoint work correctly

#### What are the relevant tickets?  

closes #31
closes #57
closes #46 
#27

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -1,71 +1,70 @@
 export const getAllMovies = async () => {
-  const response = await fetch('http://localhost:3000/api/v1/movies')
+  const response = await fetch('https://indecision-maker.herokuapp.com/api/v1/movies')
```
Comment (Atos20) [inline src/apiCalls.js:2]: I noticed that there is a new change from `HTTP` to `HTTPS`, is this related to the server being hosted in Heroku?

File: src/Components/HomePage/Homepage.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import { Homepage } from "./Homepage";
+import { MemoryRouter } from 'react-router-dom'
+import "@testing-library/jest-dom";
+
+describe("BrowsePage", () => {
+    it("User should see links to view different activity categories", () => {
+      render(<MemoryRouter><Homepage /></MemoryRouter>);
```
Comment (Atos20) [inline src/Components/HomePage/Homepage.test.js:9]: I am no sure it is this view or the linter's doing, but I think it would be better to indent line 9, 18, 23 and 32.

File: src/Components/BrowsePage/BrowsePage.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ export const BrowsePage = (props) => {
             return (
                 <li key ={i}className="hexa">
                     <div className="inner-hexagon">
-                        <img className="hexa-image" 
+                        <img className="hexa-image" data-testid='image-test'
```
Comment (Atos20) [inline src/Components/BrowsePage/BrowsePage.js:13]: I gotta start trying this data-testid. it looks quite handy.

**Review by Atos20 — APPROVED** [review]
Thanks for this addition, this will complement greatly the testing Nicole and will provide with our PRS.

File: src/Components/BrowsePage/BrowsePage.js (hunk 3 above)
Comment (nicolegooden) [reply src/Components/BrowsePage/BrowsePage.js:13]: Agreed! Makes it pretty easy to target specific elements via testing.

File: src/Components/BrowsePage/BrowsePage.test.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          name: "Jenga",
+          image: "boardURL"
+        }]);
+        await waitFor(async () => testResults = await boardResults())
+        render(<MemoryRouter><BrowsePage name={"movies"} data={testResults}/></MemoryRouter>);
+        expect(screen.getByText('Jenga')).toBeInTheDocument();
+        expect(screen.getByAltText('Jenga')).toBeInTheDocument();
+    });
+
+    it.skip("Card Games should load with a name and min / max players", async () => {
```
Comment (nicolegooden) [inline src/Components/BrowsePage/BrowsePage.test.js:58]: Should we unskip this last test?

File: src/Components/HomePage/Homepage.test.js (hunk 2 above)
Comment (nicolegooden) [reply src/Components/HomePage/Homepage.test.js:9]: Yeah I think it'd be a little more readable that way, but I know it doesn't affect the way the test functions.

File: src/apiCalls.js (hunk 1 above)
Comment (nicolegooden) [reply src/apiCalls.js:2]: Great question, Orlando! I'd love to know, too.

**Review by nicolegooden — APPROVED** [review]

File: src/Components/BrowsePage/BrowsePage.test.js (hunk 4 above)
Comment (Atos20) [reply src/Components/BrowsePage/BrowsePage.test.js:58]: I am not sure what the best practice is here but I am leaning towards un-skipping the test, so the can see the beautiful green color when they see all the test passing :)

File: src/Components/BrowsePage/BrowsePage.js (hunk 3 above)
Comment (Atos20) [reply src/Components/BrowsePage/BrowsePage.js:13]: I agree I should implement it more from now on
